app/views/world_locations/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<% page_title "UK and the world" %>
+<% page_title "Help and services around the world" %>
```

Comment:
Good detective work figuring out where this update needs to take place 🕵️‍♂️ 

My one tiny suggestion: we can see that "Help and services around the world" is repeated now on this line and on L8. So I wonder if the string `Help and services around the&nbsp;world` could be moved to the [locale file](https://github.com/alphagov/whitehall/blob/4ecfa1b4f3400a6de3cdece2148d3a5dd5a8e340/config/locales/en.yml#L577) under `world_location.content` like the rest of the content in this partial?

It could then be referenced in this partial as something like:
```suggestion
<% page_title sanitize(t("world_location.content.heading")) %>
```


For reference: `sanitize` is a method which sanitises html input in rails. I think it's being used on L8 because of the `&nbsp;` entity (they probably used `&nbsp;` here to prevent the word "world" being alone on a new line on certain breakpoints)

The `t()` bit is a translation function – it's basically a feature of the I18n API in Rails that you can use to look up translations. So let's say there if was a Welsh translation for "Help and services around the world" in the [Welsh locale file](https://github.com/alphagov/whitehall/blob/4ecfa1b4f3400a6de3cdece2148d3a5dd5a8e340/config/locales/cy.yml), then `t()` would return that translated string when the locale of the page is Welsh.

Reply:
Thanks Dana,

The HTML `&nbsp` entity is still included in the Title tag even after using the `sanitize` method.

I looked into this further and it looks like the `sanitize` method is only applied to HTML elements and attributes.

The HTML still renders the white space correctly when used in the title, I cannot see an issue with keeping the `&nbsp` entity in the title, from either an Accessibility or SEO perspective.

I tested using VoiceOver and the title was read out correctly, I also checked the guidelines with regards to whitespace and it would only fail if the whitespace was used in a word.

https://www.w3.org/TR/WCAG20-TECHS/F32.html